### ISSUE TITLE
read_mpt_cycles() does not work if .mpt file contains a single cycle:

### ISSUE THREAD

**Issue by GES-lbabetto** [GES-compchem/GES-echem-suite#26]
.mpt files which contain a single cycle are not correctly processed by the read_mpt_cycles() routine.

The program expects a line containing "Number of loops : " to read the number of cycles and a series of lines starting with "Loop " to determine the cycles position:

```
for line_num, line in enumerate(file):
    if "Number of loops : " in line:
        ncycles = int(line.split(" ")[-1])

    # Before the output of the experiment, EClab lists the
    # starting and ending line of each loop. These will be used
    # to slice the pandas dataframe into the different cycles.
    if "Loop " in line:
        loop_num = int(line.split(" ")[1])
        first_pos = int(line.split(" ")[-3])
        second_pos = int(line.split(" ")[-1])
        delims.append([loop_num, first_pos, second_pos])

    if "mode\t" in line:
        beginning = line_num
        break
```

This section is not present if the file contains a single cycle, which causes the program to not process the file correctly.

Suggested fix: after finding the line containing "mode", all the rest of the file is a single cycle, therefore it might be better to default ncycles = 1 and overwrite it only if "Number of loops : " is found. If ncycles == 1, then keep reading the file until the end.

**Comment by GES-lbabetto** [comment]
Fixed issue in pull request #27. The program now correctly reads .mpt files containing a single cycle. Functionality for multi-cycle .mpt files is unchanged.

Known issue:

- due to how the program reads the pandas table, the last point will be skipped. 

This is due to the lines `data["header"][first_row:last_row][data["ox/red"] == 1]`, where even if `last_row` is -1, the program stops at the last row, without reading it.

**Comment by GES-mpalermo** [comment]
Closed #27
